.github/workflows/sigma-validation.yml

```
@@ -35,14 +35,23 @@ jobs:
   validate-sigma-rules:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
         with:
           fetch-depth: 0
       - name: Set up Python 3.10
-        uses: actions/setup-python@v3
+        uses: actions/setup-python@v5
         with:
           python-version: "3.10"
       - name: Install dependencies
-        run: pip install check-jsonschema
+        run: pip install requests==2.31.0 check-jsonschema==0.28.0
```

I think we can use the latest versions with no issues. As there no breaking changes or hard dependencies
```suggestion
        run: pip install requests check-jsonschema
```